app/config.py

```diff
@@ -12,5 +12,7 @@ class Config:
 
     TESTING = os.getenv("TESTING", False)
     DATASET_DIRECTORY = os.getenv("DATASET_DIRECTORY", "/datasets/")
-    LOAD_IMAGES_ON_START = False
+    CREATE_CATEGORIES_FROM_FILE = os.getenv("CREATE_CATEGORIES_FROM_FILE")
```

Comment:
Is there a reason for two seperate files? I would perfer to have one with with a config property `Config.CREATE_FROM_FILE` (makes it simpler).

```javascript
{
  "categories": [
    { name: "cat", ...}
  ],
  "datasets": [
    { "name": "dataset", categories: ["a", "b", "c"] }
  ]
}
```

Reply:
no reason; one file sounds great